bin/schema-registry-stop

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-exec $(dirname $0)/schema-registry-stop-service io.confluent.kafka.schemaregistry.rest.Main
+exec $(dirname $0)/schema-registry-stop-service io.confluent.kafka.schemaregistry.rest.SchemaRegistryMain
```

Comment:
This is probably a kind of obscure compatibility issue, but if you upgraded while the service was running then the previous version would have started `io.confluent.kafka.schemaregistry.rest.Main` but this would try to kill `io.confluent.kafka.schemaregistry.rest.SchemaRegistryMain`.

Not sure we actually need to worry about this, but it is something that could happen once we publish new packages and a user upgrades via apt-get or yum while the service is running.

Maybe double check the exact semantics of `schema-registry-stop-service` and figure out if we can cover both `Main` and `SchemaRegistryMain` in one command?